Use latest avro package?

Would you accept a patch that would bump the **avro** dependency to the latest (which is 0.5.2)? Right now your _theta.cabal_ file is holding you to `< 0.4.7`.

I did a `stack build` with lts-18.21 and `>= 0.5.2` and while there were compile failures they didn't seem related to Data.Avro so using package that Alexey is now maintaining seems a good bet.

```
/home/andrew/vcs/haskell/theta-idl/theta/src/Theta/Metadata.hs:63:31: error:
    • Couldn't match type ‘[a0]’ with ‘Maybe Text.Text’
      Expected type: Test.QuickCheck.Gen.Gen (Maybe Text.Text)
        Actual type: Test.QuickCheck.Gen.Gen [a0]
```

--- a/.gitignore
+++ b/.gitignore
@@ -74,6 +74,7 @@ cabal.sandbox.config
 *.hp
 *.eventlog
 .stack-work/
+stack.yaml.lock
 cabal.project.local
 cabal.project.local~
 .HTF/

--- a/theta/stack.yaml
+++ b/theta/stack.yaml
@@ -1,0 +1,1 @@
+resolver: lts-18.21

--- a/theta/theta.cabal
+++ b/theta/theta.cabal
@@ -32,14 +32,13 @@ common shared
                , aeson
                , aeson-pretty
                , aeson-qq
-               , avro >=0.4.2.0 && <0.4.7
+               , avro >=0.5.2.0
                , binary
                , bytestring
                , containers
                , deepseq
                , filepath
                , hashable
-               , language-rust
                , megaparsec
                , mtl
                , prettyprinter

